probe-rs/src/debug/debug_info.rs

```diff
@@ -2047,4 +2049,21 @@ mod test {
 
         insta::assert_snapshot!(printed_backtrace);
     }
+
+    #[test]
+    fn test_print_stacktrace() {
+        let elf_path = Path::new("./tests/gpio-hal-blinky");
+        let mut adapter = CoreDump::load(Path::new("./tests/coredump")).unwrap();
+
+        let debug_info = DebugInfo::from_file(elf_path).unwrap();
+        let stack_frames = debug_info.unwind(&mut adapter).unwrap();
+
+        let printed_backtrace = stack_frames
+            .into_iter()
+            .map(|f| f.to_string())
+            .collect::<Vec<String>>()
+            .join("");
+
+        insta::assert_snapshot!(printed_backtrace);
+    }
```

Comment:
I can't even begin to describe how valuable this is going to be for me. Thank you!!!!